inspec/a2-api-integration/controls/notifications-alert.rb

```diff
@@ -180,6 +198,36 @@
         expect(after_request_count).to eq(before_request_count + 1)
       end
 
+      it "the webhook receives an alert with the correct non ignored failed resource" do
+        # Get the current request count on the requestbin service
+        request = inspec.http("#{requestbin_url}/#{requestbin_name}")
+        before_request_count = JSON.parse(request.body, symbolize_names: true)[:request_count]
+
+        # Create the CCR Rule
+        expect(create_ccr_rule.http_status).to eq(200)
+        sleep (5)
+
+        # Send Failed CCR
+        expect(send_failed_ccr_with_ignored_failure.http_status).to eq(200)
+        sleep (5)
+
+        # Check if the requestbin received the alert
+        request = inspec.http("#{requestbin_url}/#{requestbin_name}")
+        after_request_count = JSON.parse(request.body, symbolize_names: true)[:request_count]
+        expect(after_request_count).to eq(before_request_count + 1)
+
+        # get the last sent notification
+        notification = JSON.parse(request.body, symbolize_names: true)[:requests][before_request_count]
```

Comment:
Nitpick: instead of accessing this by indexing into the array, you could replace `[before_request_count] with `.last`.

Reply:
I always forget how friendly Ruby is with lists.